types/list.js/index.d.ts

```diff
@@ -28,7 +28,7 @@ declare class List {
     update(): void;
     reIndex(): void;
     fuzzySearch(searchString: string, columns?: string[]): void;
-    on(event: string, callback: () => void): void;
+    on(event: List.Events, callback: (list: List) => void): List;
```

Comment:
This should be singular because it refers to any one individual event. 
i.e. `List.Event` instead of `List.Events`